src/twisted/internet/interfaces.py

```diff
@@ -1508,15 +1516,15 @@ def removeSystemEventTrigger(triggerID):
             C{triggerID}.
         """
 
-    def callWhenRunning(callable, *args, **kw):
+    def callWhenRunning(callableObj, *args, **kw):
```

Comment:
I think that this should be `callable` and  the changes can be done in this PR

```
src/twisted/test/test_adbapi.py:    def callWhenRunning(self, function):
src/twisted/test/test_task.py:    def callWhenRunning(self, callable, *args, **kwargs):
src/twisted/internet/interfaces.py:    def callWhenRunning(callable, *args, **kw):
src/twisted/internet/base.py:    def callWhenRunning(self, _callable, *args, **kw):
src/twisted/internet/testing.py:    def callWhenRunning(self, callable, *args, **kw):
```

Reply:
I've moved this to: https://github.com/twisted/twisted/pull/1325